external/header.php

```diff
@@ -7,6 +7,26 @@
 
 include(dirname(__FILE__) . '/../xhprof_lib/config.php');
 
+function getExtensionName()
+{
+    if( extension_loaded( 'tideways' ) )
```

Comment:
Well, in existing code there also doesn't seem to be a 100% pattern, but i'll format it anyway